eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -188,5 +188,6 @@ if ($GetGithubIoDocIndexFn -and (Test-Path "function:$GetGithubIoDocIndexFn"))
 }
 else
 {
-    LogWarning "The function 'GetGithubIoDocIndexFn' was not found."
+    LogWarning "The function 'GetGithubIoDocIndexFn' was not found.`
+    Make sure it is present in eng/common/scripts/common.ps1 and eng/scripts/Language-Settings.ps1"
```

Comment:
It doesn't need to be in common.ps1. These functions should only need to be present in Language-Settings.ps1.

Reply:
Yes but the variable name that identifies the language specific function it is in common.ps1, so it still might be hard for someone to identify what's going on.

Reply:
The variable name might be there but most folks don't need to go there to fix a missing method, that is mostly interesting for the folks defining and using that method. 

In any rate perhaps it might be a good idea to provide a link to some documentation talking about the entire process we are using here.